INSTALL.md

```diff
@@ -30,9 +30,10 @@ If you'd like to create a VM with BLT, you will require the following additional
 
         brew tap caskroom/cask
         brew install ansible
+        brew upgrade ansible
```

Comment:
I am unsure how to do this right - i installed ansible and it gave me ansible 2.0, i had to run brew upgrade ansible to get 2.2. Is there a way to request a minimal 2.2 version?